doc_classes/TrainController.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TrainController" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/godotengine/godot/master/doc/class.xsd">
+	<brief_description>
+		Base train controller controlling all train systems
+	</brief_description>
+	<description>
+		Base train controller
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_brake" qualifiers="const">
+			<return type="TrainBrake" />
+			<description>
+				Returns TrainBrake class instance
+			</description>
+		</method>
+		<method name="get_engine" qualifiers="const">
+			<return type="TrainEngine" />
+			<description>
+				Returns TrainEngine class instance
+			</description>
+		</method>
+		<method name="get_mover_state">
+			<return type="Dictionary" />
+			<description>
+				Returns dictionary of values from iternal mover. Useful for debugging
+			</description>
+		</method>
+		<method name="get_security_system" qualifiers="const">
+			<return type="TrainSecuritySystem" />
+			<description>
+				Returns TrainSecuritySystem class
+			</description>
+		</method>
+		<method name="update_mover" qualifiers="const">
+			<return type="void" />
+			<description>
+				Updates iternal mover
```

Comment:
Można to inaczej zapisać jakoś. To metoda, która aktualizuje wewnętrzny obiekt fizyki na podstawie properties/stanu kontrolera i innych powiązanych elementów z całej kompozycji.